packages/@ourworldindata/grapher/src/stackedCharts/StackedAreaChart.stories.tsx

```diff
@@ -84,11 +84,11 @@ export const EntitiesAsSeriesWithMissingRowsNoInterpolationRelative =
     (): JSX.Element => (
         <svg width={600} height={600}>
             <StackedAreaChart
-                enableLinearInterpolation={false}
                 manager={{
                     ...entitiesChart,
                     table: table.dropRandomRows(30, seed),
                     isRelativeMode: true,
+                    enableLinearInterpolation: false,
```

Comment:
I'm not sure why this change here is not throwing TS off in CI, but it's not valid and is rightly marked as an error in VS Code.